Automattic/wp-calypso
# Extensions: Zoninator - Zones dashboard API integration

This PR aims to integrate zones dashboard with Zoninator's REST API.  
There is a loading state similar to the one on the plugins overview, but it's only triggered in case there are no zones or no previous state. Otherwise, the previous list of zones is shown and updated to reflect any recent changes - if any.

![screen shot 2017-08-02 at 11 15 08](https://user-images.githubusercontent.com/8056203/28866680-f8a2ee56-7773-11e7-956b-845805f72f5f.png)

# Testing

In order for this to work, you'll need to checkout `update/rest-api` branch of [Automattic/zoninator](https://github.com/Automattic/zoninator) on your test site and run `./scripts/build_mixtape.sh`.

- Go to zoninator settings `/extensions/zoninator` and select a site.
- Notice the loading state before the request is completed.
- If necessary, add a zone or two through WP Admin ( WP Admin > Zones ).
- Go back to the zones dashboard and ensure the new zones are showing up.
